### ISSUE TITLE
Migration fails since 0.15.0-rc7

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-desktop#13048]
## Description

My main account works just fine on `0.15.0-rc5`.
I'm able to run it with `rc6`, but any version from `rc7` crashes for me during login.

## Logs

[app_20231219_142201.log](https://github.com/status-im/status-desktop/files/13716761/app_20231219_142201.log)
[geth.log](https://github.com/status-im/status-desktop/files/13716771/geth.log)

### Additional Information

- Status desktop version: 0.15.0-rc7 and newer
- Operating System: MacOS Sonoma

**Comment by igor-sirotin** [comment]
cc @iurimatias @anastasiyaig

**Comment by anastasiyaig** [comment]
@igor-sirotin could it be shards.test  branch? we use another go branch in rc 7...

**Comment by anastasiyaig** [comment]
this one in particular https://github.com/status-im/status-go/tree/test-shard-test

**Comment by igor-sirotin** [comment]
> @igor-sirotin could it be shards.test branch? we use another go branch in rc 7...

seems like that 👍 
close the issue if it expected then

**Comment by anastasiyaig** [comment]
@richard-ramos shall the migrations work in this case?

from: 0.171.x go branch to test-shards-test branch (which is release is looking at now)

**Comment by anastasiyaig** [comment]
https://github.com/status-im/status-desktop/issues/13053 could be the same problem

**Comment by richard-ramos** [comment]
I really haven't tried to move from one branch to other with the same account. `test-shard-test` has more migrations than `release/0.171.x`.

`test-shard-test`:
![image](https://github.com/status-im/status-desktop/assets/1106587/73d34e4b-d226-45b9-be46-fdc458f68015)

`release/0.171.x`:
![image](https://github.com/status-im/status-desktop/assets/1106587/e8298e4e-1b26-4347-a48c-c631fb840f14)

Your geth.log seems to not include any interesting error, so the failure might be happening on the client side



For what is worth, I did try the following too succesfully:

1. Downloaded v0.14.1 https://github.com/status-im/status-desktop/releases/tag/0.14.1
2. Executed the AppImage with this command, and created an account: `./StatusIm-Desktop-v0.14.0-rc.4-dfab56-x86_64.AppImage --datadir=~/test-status-migration`
3. Cloned and built status desktop using this commit:
```
commit d57393bd6cdae06653574a04235c659d5273e6a9 (HEAD)
Author: Jonathan Rainville <rainville.jonathan@gmail.com>
Date:   Wed Nov 22 14:18:18 2023 -0500

    release candidate 0.15.0 RC 5
```
4. Executed it with `make run ARGS=--datadir=~/test-status-migration`. I was able to login succesfully.
5. Cloned and built status desktop using this commit:
```
commit f06706f0e06f8c80dad3ebe24e57d4459286ec7b
Author: Iuri Matias <iuri.matias@gmail.com>
Date:   Wed Dec 13 17:51:07 2023 -0500

    release candidate 0.15.0 RC 9
```
5. Executed it with `make run ARGS=--datadir=~/test-status-migration`. I was able to login succesfully.

I was also able to go from an account created with v0.14.1 directly to v0.15.0 RC9  ,  and from an account created with v0.15.0 RC5 to RC9
